### PR TITLE
Recreated the Footer , Renewed the Gallery page

### PR DESCRIPTION
**Pull request by pramodh123kit**
## Description
Recreated the Footer. Renewed the Gallery page too. @RandilaP  Please take a look and let me know if any changes are needed.
## Changes Made
Created the footer
Renewed the gallery page

## Screenshots

Capture

### CODE REVIEW

**Comment by RandilaP** [comment]
@pramodh123kit It looks great. But do we need two sections for social media. And I think gallery doesn't belong to about us section.

**Comment by pramodh123kit** [comment]
@RandilaP Ok I will remove the Gallery. I put the Socials a second time just for saturation. Else it will be just 2 , not enough.  What do you think?
Also do you think we should remove the *Sitemap* from the About us section too?

**Comment by RandilaP** [comment]
@pramodh123kit Yeah. We need something to fill. https://elixir.redcross.lk/ can u make this kinda one

**Comment by pramodh123kit** [comment]
@RandilaP  The newsletter part included?

**Comment by RandilaP** [comment]
image

**Comment by pramodh123kit** [comment]
But we can't use any of those tags in our footer right? non related

**Comment by RandilaP** [comment]
> @RandilaP The newsletter part included?

@pramodh123kit Nope. It's just a example. I thought if can't fill that we can go with that kinda template

**Comment by pramodh123kit** [comment]
@RandilaP  Ok I can change it up according to that. Should we include the Newsletter part too?

**Comment by RandilaP** [comment]
> @RandilaP Ok I can change it up according to that. Should we include the Newsletter part too?

nope. We don't have one. And after the changes please update the PR too

**Comment by pramodh123kit** [comment]
@RandilaP  Just this part then?

Capture

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```
Comment (RandilaP) [inline .vscode/settings.json]: whats this

File: Gallery/index.html [hunk 2]
```diff
@@ -22,144 +19,133 @@
...
+            </div>
           </div>
-        </div>
-      </div>
-    </section>
 
-    <!-- Gallery 3 -->
+          <div class="item">
+            <img src="images/5b7.jpg"/>
+            <div class="p5 text--container">
```
Comment (RandilaP) [inline Gallery/index.html]: what's p5 means. and change that to text-container

**Comment by RandilaP** [comment]
good job @pramodh123kit

**Pull request merged**